docs/guides/typescript.md

```diff
@@ -0,0 +1,9 @@
+---
+title: TypeScript
+---
+
+# TypeScript
+
+Remix seamlessly supports both JavaScript and TypeScript. If you name a file with a .ts or .tsx extension, it will treat it as TypeScript (.tsx is for TypeScript files [with JSX](https://www.typescriptlang.org/docs/handbook/jsx.html) in them). But it isn't required. You can write all your files as .js files if you don't want TypeScript.
+
+Right now it will automatically compile any .ts and .tsx files in the app directory. In the starter-express repo, it also supports .ts and .tsx files in the loaders directory, though we just use tsc to compile them directly instead of our compiler. This will probably change soon.
```

Comment:
There’s no loaders directory anymore, only the app one

Reply:
@sergiodxa I have removed the loaders directory part.